packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -141,10 +144,16 @@ class KnexListAdapter extends BaseListAdapter {
     this.tableName = this.key;
   }
 
-  prepareFieldAdapter(fieldAdapter) {
-    if (!(fieldAdapter.isRelationship && fieldAdapter.config.many)) {
-      this.realKeys.push(...(fieldAdapter.realKeys ? fieldAdapter.realKeys : [fieldAdapter.path]));
-    }
+  prepareFieldAdapter() {}
```

Comment:
Should we console log here that this method is deprecated?

Reply:
The method's not deprecated, it's just that this adapter happens to not use it, which is fine.